submissions/rafiatAminu/DOM/product.js

```diff
@@ -0,0 +1,76 @@
+//const post = document.querySelectorAll(".post");
+//console.log(post[0]);
+//work2.innerText = "Hello world";
+//post[0].innerHTML = "water";
+
+//for (let i=0; i<post.length; i++){
+// //    post[i].style.color = "";
+// //post.forEach(Element => {
+// //    Element.style.color= "turquoise";
+// //});
+
+// //const secondsss = document.getElementById("secondss");
+// //console.dir(secondsss);
+
+// //let container = document.querySelector(".container");
+// //newSpan = document.createElement("span");
+// //.innerHTML = "Spantastic";
+// //newSpan.style.color = "green";
+
+// //console.log(container);
+// //container.appendChild(newSpan);
+
```

Comment:
is there a reason for the comments above?

Reply:
No, that’s just me practicing. @adaugochi.